awesome-tray.el

```diff
@@ -1200,10 +1200,12 @@ If right is non nil, replace to the right"
   "Get new text to be displayed."
   (interactive)
   (let* ((tray-info (awesome-tray-build-active-info))
-         (minibuffer-info (current-message))
+         (minibuffer-info (or (current-message) ""))
+         (minibuffer-info (set-text-properties 0 (length minibuffer-info) nil minibuffer-info))
+         (minibuffer-info (if (stringp minibuffer-info) minibuffer-info ""))
          (blank-length (- (awesome-tray-get-frame-width)
                           (string-width tray-info)
-                          (string-width (if minibuffer-info minibuffer-info "")))))
+                          (string-width (or (cdr (split-string minibuffer-info "\n")) "")))))
```

Comment:
If you want pick last line of minibuffer, you should use `(car (last (split-string minibuffer-info "\n")))` why `cdr` ?